lib/task-data/base-tasks/copy-key.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+
+module.exports = {
+    friendlyName: 'Copy Key',
+    injectableName: 'Task.Base.Ssh.CopyKey',
+    runJob: 'Job.CopyKey',
+    requiredOptions: [],
```

Comment:
@heckj yep, looks like. It's how we do it in other jobs as well, though I think we could re-design this when we refactor base-tasks (maybe have a schema option for required target).